index.js

```diff
@@ -0,0 +1,58 @@
+function validatePassword(password) {
+  return (password) &&
+    (isValidLength(password)) &&
```

Comment:
You don't need the extra parentheses. I like the first check for just `password`, it's what we call `defensive programming`. If password came in as undefined then `password.length` would throw an error. This prevents that.

```JavaScript
function validatePassword(password) {
  return password &&
    isValidLength(password) &&
    isLowerCase(password) &&
    isUpperCase(password) &&
    isNumber(password) &&
    isSpecial(password)
}
```